server/store/sqlstore/store.go

```diff
@@ -233,6 +233,10 @@ func (s *SQLStore) Init() error {
 		return err
 	}
 
+	if err := s.addColumn(subscriptionsTableName, "syncNeeded", "BOOLEAN"); err != nil {
```

Comment:
To confirm, is this `NULLABLE`? Do we handle this with the requisite `COALESCE` when querying this table? Are all existing queries explicit in which columns they return, i.e. no `SELECT *` anywhere that would break a previous version of the plugin?

Reply:
Yes, all the queries are explicit about what they return. Probably we should make it not null and default false, let me try to fix that.

Reply:
I change it to BOOLEAN NOT NULL DEFAULT False